CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+# Contributing Guidelines
+---
+
+
+Firstly thanks for your contributions!!! :sparkling_heart::sparkling_heart:
+
+If you have resources you would like to share from your event or resources you think our community
```

Comment:
I am not sure if we need this sentence, not much helpful. What do you think @nverbic

Reply:
Yes, I agree. I will remove this line.